templates/compositions/ec-component-content-item/content-item.html.twig

```diff
@@ -0,0 +1,123 @@
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<article class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% if _content_item.image and _content_item.image_position == 'before' %}
+    <div role="img" aria-label="{{ _content_item.image.alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-lg-block"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+    <div role="img" aria-label="{{ _content_item.image.alt }}"
+         class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-mr-xl ecl-u-d-none ecl-u-d-lg-none"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
+    </div>
+  {% endif %}
+  <div class="ecl-u-flex-grow-1">
+    {% if _content_item.meta is not empty %}
+      <div class="ecl-content-item__meta ecl-u-type-s ecl-u-type-color-grey-75">{{ _content_item.meta|join(" | ") }}</div>
+    {% endif %}
+    <div class="ecl-content-item__title ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+      {%- if _content_item.title.path %}
+        {% include '../ec-component-link/link.html.twig' with {
+          link: _content_item.title
+        } only %}
+      {% else %}
+        {{- _content_item.title.label }}
+      {%- endif -%}
+    </div>
+    <div class="ecl-content-item__description ecl-u-type-paragraph ecl-u-type-color-grey-100 ecl-u-mt-xs">{{ _content_item.description }}</div>
+    {% if _content_item.infos is defined and _content_item.infos is not empty and _content_item.infos is iterable %}
+      <ul class="ecl-content-item__info ecl-u-mt-m ecl-unordered-list ecl-unordered-list--no-bullet">
+        {% for info in _content_item.infos %}
+          <li class="ecl-content-item__info-item ecl-u-d-flex ecl-u-align-items-center ecl-u-type-color-grey-75 ecl-unordered-list__item">
+            {% include '../ec-component-icon/icon.html.twig' with {
+              icon: info.icon|merge({ size: 'm' })
+            } only %}
+            <span class="ecl-u-type-s ecl-u-ml-s">{{ info.label }}</span>
+          </li>
+        {% endfor %}
+      </ul>
+    {% endif %}
+  </div>
+  {% if _content_item.image and _content_item.image_position == 'after' %}
+    <div role="img" aria-label="{{ _content_item.image.alt }}" class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-ml-xl ecl-u-d-lg-none"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:5rem;width:7.5rem"></div>
+    <div role="img" aria-label="{{ _content_item.image.alt }}"
+         class="ecl-u-ratio-3-2 ecl-u-flex-shrink-0 ecl-u-ml-xl ecl-u-d-none ecl-u-d-lg-block"
+         style="background-image:url(&quot;{{ _content_item.image.src }}&quot;);background-size:contain;height:8.75rem;width:13.125rem">
+    </div>
```

Comment:
The image part is used twice, here and on the top. We can use [Twig's `macros`](https://twig.symfony.com/doc/2.x/tags/macro.html) to define this part only once and use it in two different places.

Reply:
The part image is not really the same, some classes are different.
Updated.